ui/src/app/pages/profile/demographic-survey.tsx

```diff
@@ -112,21 +117,28 @@ export class DemographicSurvey extends React.Component<Props, State> {
 
     const errors = this.validateDemographicSurvey(demographicSurvey);
 
-    return <div style={{marginTop: '1rem', paddingLeft: '3rem', width: '26rem'}}>
+    return <div style={{marginTop: '1rem', paddingLeft: '3rem', width: '32rem'}}>
       <TextColumn>
         <div style={{fontSize: 28, fontWeight: 400, marginBottom: '.8rem'}}>Optional Demographics Survey</div>
-        <div style={{fontSize: 16, marginBottom: '.5rem'}}>
-          Please complete Step {requireInstitutionalVerification ? '3 of 3' : '2 of 2'}
-        </div>
-        <div>
+        {this.props.showStepCount &&
+          <div style={{fontSize: 16, marginBottom: '.5rem'}}>
+            Please complete Step {requireInstitutionalVerification ? '3 of 3' : '2 of 2'}
+          </div>
+        }
+        <div style={{
+          backgroundColor: colorWithWhiteness(colors.accent, .85),
```

Comment:
More of a lou question and Not part of this story (we can discuss this in UI meeting) but should the background color be consistent with Publicly displayed icon i.e colors.accent with 0.75

Reply:
Yeah, it probably should given the background image issue